src/test/java/com/gildedrose/GildedRoseTest.java

```diff
@@ -19,4 +19,31 @@ void fooTest() {
         assertEquals(40, app.items[0].quality);
     }
 
+
+   @Test
+   void agedBrieTest() {
+       Item[] items = new Item[] { new Item("Aged Brie", 5, 0) };
+       GildedRose gildedRose = new GildedRose(items);
+       gildedRose.updateQuality();
+
+       assertEquals("Aged Brie", gildedRose.items[0].name);
+       assertEquals(4, gildedRose.items[0].sellIn);
```

Comment:
it's nice to have a test scenario with one reason to fail. It's testing a unitary action. I would personally move the sellIn to another test. Or at least make the name more explicit.

But preferably one reason to fail/ test

Reply:
Done